mono/metadata/mono-debug-debugger.h

```diff
@@ -20,7 +20,7 @@ mono_debug_get_seq_points (MonoDebugMethodInfo *minfo, char **source_file, GPtrA
 MONO_API void
 mono_debug_free_locals (MonoDebugLocalsInfo *info);
 
-MONO_API void
```

Comment:
I was just following `mono_debug_free_locals`, hence added MONO_API in first place...